src/twisted/web/newsfragments/10096.bugfix

```diff
@@ -0,0 +1 @@
+skip failing twisted.web.test.test_http.QueryArgumentsTests.testParseqs
```

Comment:
I think that this should be an .ignore as the test code is internal stuff and end users shouln't care... but this is minor comment.